xbmc/settings/SettingsComponent.cpp

```diff
@@ -100,12 +97,10 @@ bool CSettingsComponent::Load()
   }
 }
 
-void CSettingsComponent::Deinit()
+void CSettingsComponent::Uninitialize()
```

Comment:
Deinitialize is the more popular synonym

Reply:
Okay, will change. Just thought changing from "De" to "Un" would be fine as we use `Uninitialize` all over our code base.